Homeworks

vaal1212 opened the pull request.

vaal1212 committed:
Merge pull request #1 from vaal1212/develop

Removing secret key from setting.

Commit:
1. db app created
2. Task model created and migrated
3. Task model registered in admin

Commit:
Time zone changed to Ukraine/Kyiv EET

Commit:
1. How to start Jupyter
2. Test notebook

Commit:
1. Installed postgreSQL

2. Installed psycopg2
3. Created DB in postgres
4. Changed setting.py, so default DB is postgres\itea_crossfit_project_db1
	DB pass is in .env file.
5. Migrated Django - success
6. Created superuser
6. Runserver executes (admin works)
7. Cleaning of gitignore
8. Adding pipfile - created repository in the directory of the django project, so unfortunately pip file is 1 directory above and is not tracked. Will periodically copy it to _copy_of_pipfile directory. Lesson learned.

vaal1212 commented:
Please ignore ipynb files - those are from the lesson commit I created while trying jupyter

stanislavio commented on sportera_crossfit/settings.py:
```diff
@@ -80,10 +87,30 @@
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
All the data about your database replace into .env file. HOST, PORT, USER, PASSWORD, DBNAME

stanislavio commented on sportera_crossfit/settings.py:
```diff
@@ -80,10 +87,30 @@
+#Portable postgres: https://garethflowers.dev/postgresql-portable/
+
+#SQL to execute in PostGres: 
+#   CREATE DATABASE itea_crossfit_project_db1;
```
Цю інформацію можна винести у файл ReadME. Типу зробити розділ setup project і в ньому описати всю процедуру яку потрібно пройти для підняття проекту локально

Commit:
Changes as per pull request review comments of 02Aug2022.

1. Moved all DB parameters to .env file
2. Moved help on postgres to README.txt

Commit:
Changes as per pull request review comments of 02Aug2022.

1. Moved all DB parameters to .env file
2. Moved help on postgres installation to README.txt

vaal1212 reviewed (commented):
Додав зміни які потрібні
